src/hooks/useMe.ts

```diff
@@ -21,9 +20,8 @@ interface MeResponse {
 
 export function useMe() {
   const { httpClient } = useHttpClient();
-  const { lastAcceptedId } = usePendingInvite();
 
-  const useMeQuery = useQuery(['fhir/dstu3/$me', lastAcceptedId], () =>
```

Comment:
@bruce-glazier You added this. I think it's no longer needed, since we're going to clear the QueryClient entirely when accepting a new invite now, thanks to this line: https://github.com/lifeomic/react-native-sdk/pull/488/files#diff-c7bf47785b64fa6a95226baae705ac2b51d046cd1846c3c02fca15b17d7a6733R77-R80

Would love a sanity check there.

Reply:
This seems correct to me.